### DISCUSSION TITLE
Issue while deploying module template to Globe

### DISCUSSION THREAD

**Discussion by ps6067966** [serverpod/serverpod#3097] [Q&A]
![image](https://github.com/serverpod/serverpod/assets/26248740/df8ebbfe-8347-44ea-b354-f0dc376a9a99)

Please help.

**Comment by Isakdl** [comment]
This means that the Serverpod command is not able to find the client project at the path: `../project_name_client`.

Make sure that the client project exists at that path, or configure a different path in the `generator.yaml` file. The client project is required to run the code generator.

**Comment by ps6067966** [comment]
> This means that the Serverpod command is not able to find the client project at the path: `../project_name_client`.
> 
> Make sure that the client project exists at that path, or configure a different path in the `generator.yaml` file. The client project is required to run the code generator.

New project. 
And i can see client project generated by serverpod. 

And i can see path set by serverpod. 

Then why this error?

**Comment by Isakdl** [comment]
Are you getting this error locally or only in a CI environment? 

If it is only in the CI env, you are probably not uploading the client.
